CDCgov/prime-reportstream
# 13798: additional obx mappings for Radxmars

This PR adds the additional components to the OBX->Observation->OBX mapping to match the spec

Test Steps:
There is an automated test that covers this
1. Send this message with the MARS ignore sender
```
MSH|^~\&|MMTC.PROD^2.16.840.1.113883.3.8589.4.2.106.1^ISO|CAREEVOLUTION^00Z0000024^CLIA|AIMS.INTEGRATION.PRD^2.16.840.1.114222.4.3.15.1^ISO|AIMS.PLATFORM^2.16.840.1.114222.4.1.217446^ISO|20240404110305+0000||ORU^R01^ORU_R01|20240404110305_433848dc33ca4ce89ff4773fed48e8f6|P|2.5.1|||NE|NE|||||PHLabReport-NoAck^ELR251R1_Rcvr_Prof^2.16.840.1.113883.9.11^ISO
SFT|CAREEVOLUTION|2022|MMTC.PROD|16498||20240402
PID|1||f7aa0855e93244f19cb10feda529cc41^^^MMTC.PROD&2.16.840.1.113883.3.8589.4.2.106.1&ISO^PI||DeIdentified^DeIdentified^DeIdentified||19800101|M||2106-3^White^HL70005^^^^2.5.1|DeIdentified^^DeIdentified^VA^22043^USA||^^PH^^^555^1111234~^^Internet^DeIdentified|||||||||N^Not Hispanic or Latino^HL70189^^^^2.5.1
ORC|RE||^MMTC.PROD^2.16.840.1.113883.3.8589.4.2.106.1^ISO||||||||||||||||||SA.OTCSelfReport|^^^VA^22043
OBR|1||^MMTC.PROD^2.16.840.1.113883.3.8589.4.2.106.1^ISO|94558-4^SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by Rapid immunoassay^LN^^^^2.71|||20240404120000-0400|||||||||||||||20240404120000-0400|||F
OBX|1|CWE|94558-4^SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by Rapid immunoassay^LN^^^^2.71||260415000^Not detected^SCT^^^^20200901||||||F||||00Z0000042||BinaxNOW COVID-19 Antigen Self Test_Abbott Diagnostics Scarborough, Inc._EUA^^99ELR^^^^Vunknown||20240404120000||||SA.OTCSelfReport^^^^^&2.16.840.1.113883.3.8589.4.1.152&ISO^XX^^^00Z0000042
NTE|1||note
OBX|2|NM|35659-2^Age at specimen collection^LN^^^^2.71||39|a^year^UCUM^^^^2.1|||||F||||00Z0000042||||||||SA.OTCSelfReport^^^^^&2.16.840.1.113883.3.8589.4.1.152&ISO^XX^^^00Z0000042|||Reason|1^Foo^LN|1^Reason^LN|QST
SPM|1|^433848dc33ca4ce89ff4773fed48e8f6&MMTC.PROD&2.16.840.1.113883.3.8589.4.2.106.1&ISO||697989009^Anterior nares swab^SCT^^^^20200901|||||||||||||20240404120000-0400|20240404120000-0400    
```
2. Check that the receiver gets the message with additional fields 


## Changes
- Adds the updated OBX mappings
- Small refactor to the integration test running to support different message profiles

## Checklist

### Testing

### Process

## Linked Issues
- #13798 

## To Be Done


## Specific Security-related subjects a reviewer should pay specific attention to
